MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -304,6 +315,52 @@ protected virtual ItemUpdateType BeforeSaveInternal(TItemType item, bool isFullR
                 updateType |= ItemUpdateType.MetadataImport;
             }
 
+            // Cleanup extracted files if source file was modified
+            var itemPath = item.Path;
+            if (!string.IsNullOrEmpty(itemPath))
+            {
+                var info = FileSystem.GetFileSystemInfo(itemPath);
```

Comment:
Can't item path be a folder if it's a dvd or bluray?

Reply:
Yes, but that's handled properly by `FileSystem.GetFileSystemInfo` and later on we only do the size check if it is not a DVD or BD (those are always folders).